Added CKEditor

imanityson opened the pull request:
Added CKEditor to message and about me forms.

imanityson committed:
New charts html

I made a new chart html file. The java script in the file accesses google charts library and creates a pie chart that shows how many messages each person has sent.

imanityson committed:
Merge branch 'master' into imani

imanityson committed:
Added Text Editor to messages

Added CKEditor to messages and about me sections.

kassidywebb reviewed (approved):
It looks like you also need to make a .gitignore file so only the important changes show

imanityson committed:
Added Servlet for Create New Event

Preliminary Servlet for Create New Event. It only requests the different categories needed to store.

imanityson committed:
Deleted chart html
